### ISSUE TITLE
all: add more github status checks 

### ISSUE THREAD

**Issue by fexolm** [PieselBois/kfulint#25]
we may add status checks with gometalinter output

**Comment by Quasilyte** [comment]
You mean including gometalinter into CI?

**Comment by fexolm** [comment]
@Quasilyte no. I mean find github app, that will run gometalinter on our code.

**Comment by Quasilyte** [comment]
Ah, I see.

**Comment by weeellz** [comment]
There is no github apps with enough checks(or free), so i'll add gometalinter and goreportcard in Travis.

**Comment by Quasilyte** [comment]
> so i'll add gometalinter and goreportcard in Travis.

@weeellz, that's quite brutal.
I've seen some minor issues like typos in comments like this:
```go
// WarkingKind describes checker warning category.
// Useful for checkers that can find different kinds of issues.
//
// Should be human-readable, not cryptic.
type WarningKind string
```
Note `WarkingKind` (typo).

Our build will break if someone will not fix all of them.
Could you fix these places along with CI script changes?

**Comment by weeellz** [comment]
Ok, i'll do it.

**Comment by Quasilyte** [comment]
Just a remainder: everything under `testdata` should not be linted during CI.
It's checked by `gofmt -s -d` now, but really shouldn't.

**Comment by weeellz** [comment]
@Quasilyte, [https://github.com/alecthomas/gometalinter#whats-the-best-way-to-use-gometalinter-in-ci](url) i don't quite understand second problem.

**Comment by weeellz** [comment]
Done in #28.
